docs/architecture/adr-112-proposer-based-timestamps.md

```diff
@@ -329,6 +356,12 @@ This skew will be bound by the `PRECISION` value, so it is unlikely to be too la
 
```

Comment:
On line above, if such a situation currently exists on a chain, its community may decide to keep BFT Time (by not activating PBTS via its enable height) until the problem is fixed: BFT Time will let the chain make progress, by adding one millisecond to each new block; so if the average block time is > 1ms, the chain's time will eventually catch up.

Reply:
Yes, this [comment](https://github.com/cometbft/cometbft/pull/2223#discussion_r1480350702), right?

Reply:
Added this "workaround" to the description, while improving the description of this "pause".

Reply:
Added an action item to #2063 to address this "workaround" in the documentation.

Reply:
👍